pkg/operator/api/api.go

```diff
@@ -57,19 +57,17 @@ var clusterAccessRules = []v1beta1.PolicyRule{
 
 type Cluster struct {
 	context   *clusterd.Context
-	placement k8sutil.Placement
-	Name      string
 	Namespace string
+	placement k8sutil.Placement
 	Version   string
 	Replicas  int32
 }
 
-func New(context *clusterd.Context, name, namespace, version string, placement k8sutil.Placement) *Cluster {
+func New(context *clusterd.Context, namespace, version string, placement k8sutil.Placement) *Cluster {
 	return &Cluster{
 		context:   context,
-		placement: placement,
-		Name:      name,
```

Comment:
How about adding the `name` back and just set it the same as the `Namespace`. Later, if we decide to support multiple cluster per namespace, it would be easier.

Having a name (or id) as identifier makes more sense when building the API.